hotspot/src/os/linux/vm/cgroupSubsystem_linux.hpp

```diff
@@ -57,7 +57,7 @@ typedef char * cptr;
 
 class CgroupController: public CHeapObj<mtInternal> {
   public:
-    virtual char *subsystem_path();
+    virtual char *subsystem_path() = 0;
```

Comment:
I wonder if this will fall foul of the pre C++11 ARM32 compiler problem that 
https://github.com/openjdk/jdk8u-dev/pull/127 had

Reply:
Maybe. Until we see issues with it, I'd leave the patch as is.